### DISCUSSION TITLE
EDA

### DISCUSSION THREAD

**Discussion by AngelaSherina** [teamalgoritma/community#311] [Q&A] [answered]
saya mau tanya untuk mendapatkan hasil no 4 itu seperti apa ya? sudah menggunakan dropna & fillna namun tidak mendapatkan hasil sesuai pilihannya
![image](https://user-images.githubusercontent.com/112871237/190946388-eea1cbd4-063e-4a0c-8938-b6d32ed05987.png)
![image](https://user-images.githubusercontent.com/112871237/190946355-cdbdd6be-be9e-451b-99e4-6ef81c898596.png)
terima kasih

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @AngelaSherina , 

Pertanyaan nomor 4 meminta Anda untuk mencari data hasil aggregasi yang menampilkan nilai `median` dari `raisedAmt` untuk tiap `category`. Hasil output Anda masih menampilkan data yang belum diaggregasi (hasil aggregasi akan menampilkan 1 nilai median untuk 1 category. Silahkan eksplor kembali fungsi yang harus digunakan untuk mendapatkan nilai `median` dari `raisedAmt` per `category` tersebut.

Semoga informasi ini bermanfaat.
